vtrace/platforms/linux.py

```diff
@@ -24,6 +24,8 @@
 
 from ctypes import *
 import ctypes.util as cutil
+from vtrace.platforms.posix import PT_CONTINUE
+from ptrace.binding.func import PTRACE_EVENT_EXEC, PTRACE_O_TRACEEXEC
```

Comment:
Actually, it looks like these are defined below! any reason to get them from the ptrace module here?